templates/workloads/partials/cf/envvars.yml

```diff
@@ -19,6 +19,10 @@ Environment:
 - Name: COPILOT_MOUNT_POINTS
   Value: '{{jsonMountPoints .Storage.MountPoints}}'
 {{- end}}{{end}}
+{{- if .Publish}}{{- if .Publish.Topics}}
+- Name: COPILOT_PUBLISHERS
+  Value: '{{jsonPublishers .Publish.Topics}}'
```

Comment:
wow i didn't realize we have such an amazing macro func

Reply:
samesies here, is it possible to rename the template function to `jsonSNSTopics`

Reply:
Adjusted!